src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,195 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
```

Comment:
The import statement for `java.util.LinkedList` is unnecessary since you are implementing your own linked list. Consider removing this import to avoid confusion.